tests/foreman/cli/test_realm.py

```diff
@@ -88,7 +93,7 @@ def test_positive_realm_info_name(
             'locations': [loc.read().name for loc in module_fake_proxy.location],
         }
     )
-    request.addfinalizer(lambda: realm_cleanup(realm['id']))
+    request.addfinalizer(lambda: target_sat.cli.Realm(realm['id']).delete())
     info = Realm.info({'name': realm['name']})
```

Comment:
lots of mixes between target_sat's Realm and the default Realm. This could lead to issues

Reply:
I think @JacobCallahan is referring to the fact that this is a destructive test, but I don't think you're deleting the default realm here?

That being said, this test function will be moving with #9697 and you'll have to rebase on top of it.

Reply:
destructive test move was merged, @sambible time to rebase - some of these changes will have moved to new modules.